storage.js

```diff
@@ -0,0 +1,31 @@
+const formStorage = document.getElementById('form');
+const nameStorage = document.getElementById('name');
+const emailStorage = document.getElementById('email');
+const messageStorage = document.getElementById('message');
```

Comment:
- [ ] Great job adding the local storage feature to your portfolio! :+1:  However, please ensure that the ids for the `form`, `name`, `email`, and `message` elements are in your index.html file. If they are missing, the feature may not function as intended.